airbyte-migration/src/main/resources/migrations/migrationV0_17_0/airbyte_config/destination_definitions.yaml

```diff
@@ -0,0 +1,35 @@
+- destinationDefinitionId: a625d593-bba5-4a1c-a53d-2d246268a816
+  name: Local JSON
+  dockerRepository: airbyte/destination-local-json
```

Comment:
what if instead of copying both of these files we just create a set with all of the `dockerRepository` in `MigrationV0_17_0`? since the tag is 0.2.0 for all of these, really the only unique piece of information you're pulling out of here is that docker image name. it seems like we're copying a lot info we don't need.

e.g. `Set<String> KNOWN_IMAGES = Sets.newHashSet("airbyte/destination-local-json", "airbyte/destination-csv", etc...)`